src/ducks/settings/TogglePane.jsx

```diff
@@ -26,12 +27,22 @@ class TogglePane extends Component {
   }
 
   render() {
-    const { rows, settingsCollection, settingsKey, t, title } = this.props
+    const {
+      rows,
+      settingsCollection,
+      settingsKey,
+      t,
+      title,
+      description
+    } = this.props
```

Comment:
This is starting to get ugly, I think we should be able to refactor to have less properties here.